server/app.ts

```diff
@@ -58,6 +59,9 @@ export default function createApp(services: Services): express.Application {
     }),
   )
   app.use(setUpCurrentUser())
+
+  app.use(maintenancePageRoute())
```

Comment:
Adding it here specifically so that DPS components (header/footer) have loaded but before any other services are called. E.g. the next line - `populateSelectedEstablishment()` needs orchestration/visit scheduler to be working...